src/main/java/ch/njol/skript/expressions/ExprServerTickRate.java

```diff
@@ -0,0 +1,116 @@
+/**
+ *   This file is part of Skript.
+ *
+ *  Skript is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  Skript is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with Skript.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * Copyright Peter Güttinger, SkriptLang team and contributors
+ */
+package ch.njol.skript.expressions;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.bukkitutil.ServerUtils;
+import ch.njol.skript.classes.Changer.ChangeMode;
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Examples;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.RequiredPlugins;
+import ch.njol.skript.doc.Since;
+import ch.njol.skript.lang.Expression;
+import ch.njol.skript.lang.ExpressionType;
+import ch.njol.skript.lang.SkriptParser.ParseResult;
+import ch.njol.skript.lang.util.SimpleExpression;
+import ch.njol.util.Kleenean;
+import ch.njol.util.coll.CollectionUtils;
+import org.bukkit.event.Event;
+import org.jetbrains.annotations.Nullable;
+
+@Name("Server Tick Rate")
+@Description({
+	"Gets or sets the current tick rate of the server. The tick rate is the number of game ticks that occur in a second. Higher values mean the game runs faster.",
+	"The server's default tick rate is 20."
+})
+@Examples({
+	"send \"%server's tick rate%\" to player",
+	"set server's tick rate to 20 # This is the default tick rate.",
+	"add 5 to server's tick rate",
+	"remove 2 from server's tick rate"
+})
+@Since("INSERT VERSION")
+@RequiredPlugins("Minecraft 1.20.4+")
+public class ExprServerTickRate extends SimpleExpression<Float> {
```

Comment:
Is there any connection between this and tps? I can see them getting mixed up.

Reply:
The already existing TPS expression gets the TPS over a period of time (1, 5, 15 minutes)

This one gets the current tick rate, which is most likely more precise then the already existing expression, its also more specific to tick rate management then the TPS expression, as opposed to average TPS over intervals.

though maybe this should be an option of that expression instead of something entirely new? Love to hear other peoples thoughts

Reply:
Though, if it were to get added to that expression, I feel it would be tricky to come up with good syntax for it as `[the] tps` already exists in that expression, and returns all of the tps over the intervals.

putting this into that expression would output some dodgy syntax i feel like, I personally think it'd be best to keep them separate, but like I said before, would love to hear some other peoples thoughts

Reply:
No this doesn't get a more precise tps
It gets/sets the *goal* tps, which by default is 20. Your server can be having 10 tps but be targetting 20 tps, therefore lagging. It can have 5 tps and target 5 tps, therefor not lagging. TPS would return 10 and 5 respectively, but this expression would return 20 and 5.

Reply:
> No this doesn't get a more precise tps It gets/sets the _goal_ tps, which by default is 20. Your server can be having 10 tps but be targetting 20 tps, therefore lagging. It can have 5 tps and target 5 tps, therefor not lagging. TPS would return 10 and 5 respectively, but this expression would return 20 and 5.

whoops 😅